lib/index.js

```diff
@@ -152,8 +152,7 @@ export default {
         const cmd = [this.executablePath, 'analyze',
           '-f json',
           makeEngineString(linterEnabledEngines),
-          `'${atom.project.relativize(filePath)}'`,
-          '< /dev/null'].join(' ');
```

Comment:
I'm not sure I understand why we had this `/dev/null` business previously, but I'm also not sure that it's hurting. It's been there all along, since: https://github.com/AtomLinter/linter-codeclimate/commit/1f78f75235029c22ad19eaa935d65f84830a6e1d

The CLI doesn't currently read anything from STDIN